lib/utils/configReader.js

```diff
@@ -1,85 +1,86 @@
 'use strict'
 
 var path = require('path')
-
-var extend = require('lodash/object/extend')
+var warn = require('debug')('risingstack/trace:warn')
```

Comment:
then please add it to log by default, otherwise nobody will see :disappointed: 
@gergelyke ?